main/app_main.c

```diff
@@ -53,12 +54,17 @@ const static char http_jpg_hdr[] =
 const static char http_pgm_hdr[] =
         "Content-type: image/x-portable-graymap\r\n\r\n";
 const static char http_stream_boundary[] = "--123456789000000000000987654321\r\n";
+const static char http_bitmap_hdr[] =
+        "Content-type: image/bitmap\r\n\r\n";
 
 static EventGroupHandle_t wifi_event_group;
 const int CONNECTED_BIT = BIT0;
 static ip4_addr_t s_ip_addr;
 static camera_pixelformat_t s_pixel_format;
 
+#define CAMERA_PIXEL_FORMAT CAMERA_PF_RGB565
+#define CAMERA_FRAME_SIZE CAMERA_FS_QQVGA
```

Comment:
how about integrate Jeroen's httpd to this demo, It is quite small and easy to use
https://github.com/tuanpmt/esp32-cam-demo/blob/feature/httpd/main/app_main.c#L117